steps/qubo.py

```diff
@@ -1,6 +1,8 @@
+import json
```

Comment:
rm unnessesary import
(hey pre-commit would have caught this :P )

Reply:
Feel free to desrcribe how to use pre-commit hooks in out new guidelines ✨